src/main/steps/application-sent-for-review/content.ts

```diff
@@ -0,0 +1,27 @@
+import dayjs from 'dayjs';
+import advancedFormat from 'dayjs/plugin/advancedFormat';
+
+import { TranslationFn } from '../../app/controller/GetController';
+
+import type { CommonContent } from '../../steps/common/common.content';
+
+dayjs.extend(advancedFormat);
+
+const en = ({ partner }: CommonContent) => ({
+  title: `Your answers have been sent to your ${partner} to review`,
+  sentTo: `Your answers have been sent to your ${partner} at the following email address:`,
+  theyNeedToReviewBy: 'They need to review them and provide some of their own information by',
+  reviewDate: dayjs().add(2, 'weeks').format('MMMM Do YYYY'),
```

Comment:
Does the `Do` need to be `Dd`?

Reply:
Also will this review date remain fixed - not change if the used somehow refreshes the page etc?

Reply:
The [format](https://day.js.org/docs/en/display/format) matches the prototype which is:

Format | Output | Description
-- | -- | --
`MMMM` | January-December | The full month name
`Do` | 1st 2nd ... 31st | Day of Month with ordinal
`YYYY` | 2018 | Four-digit year

That's true that `add(2, 'weeks')` will just be today plus 2 weeks. I've updated as we will need to record when the application was submitted in [NFDIV-696](https://tools.hmcts.net/jira/browse/NFDIV-696).